src/graphql/resolvers/ssb.js

```diff
@@ -1,12 +1,21 @@
 const pull = require('pull-stream')
 const pullParaMap = require('pull-paramap')
-const { where, type, descending, toPullStream } = require('ssb-db2/operators')
+const { where, type, descending, toPullStream, votesFor, contact } = require('ssb-db2/operators')
 const { promisify: p } = require('util')
+const toSSBUri = require('ssb-serve-blobs/id-to-url')
 const SSB = require('../../ssb-server')
 
 module.exports = function Resolvers () {
   const ssb = SSB()
 
+  const ssbPort = ssb.config.serveBlobs && ssb.config.serveBlobs.port
+
+  /**
+   * Gets the profile (about) of the given profile id.
+   * NOTE: only returns a profile for those who have
+   * opted in to publicWebHosting
+   * @param {string} id - id of a profile
+   */
   const getProfile = async (id) => {
     const profile = await p(ssb.aboutSelf.get)(id)
```

Comment:
we may want to make a cache on this soon. watch the response time on loading followers to see if it's needed